### ISSUE TITLE
warning "react-dates > react-with-styles@3.2.1" has unmet peer dependency "react-with-direction@^1.1.0".

### ISSUE THREAD

**Issue by joanjoantan** [airbnb/react-dates#1348]
With react (^16.2.0):

package.json
```
"react": "^16.2.0",
    "react-accessible-accordion": "^2.4.4",
    "react-collapsible-component": "^1.1.0",
    "react-cookie": "^2.1.2",
    "react-dates": "^16.3.6",
    "react-debounce-input": "^3.2.0",
    "react-dev-utils": "lendingworks/react-dev-utils",
    "react-dom": "^16.2.0",
    "react-idle-timer": "^4.0.8",
    "react-loading-placeholder": "^1.0.6",
    "react-moment": "^0.7.0",
    "react-moment-proptypes": "^1.5.0",
    "react-paginate": "^5.2.1",
    "react-redux": "^5.0.6",
    "react-render-html": "^0.6.0",
    "react-router": "^4.2.0",
    "react-router-dom": "^4.2.2",
    "react-router-redux": "^4.0.8",
    "react-stripe-elements": "^1.6.0",
    "react-tooltip": "^3.4.0",```

**Comment by ljharb** [comment]
Since we updated from react-with-styles v2 to v3 in v16.4 / #1041, and that brought along a new peer dep requirement, that was a breaking change (cc @lencioni @majapw). I'll add it as a dep + peerDep in both master and in a v16 backport, to fix.

**Comment by ljharb** [comment]
Published v16.7.1.

**Comment by majapw** [comment]
Whoops! Thank you @ljharb
